ci/prow/lint

```diff
@@ -2,4 +2,13 @@
 
 set -euxo pipefail
 
+# Generate code and compare with the current source.
+# Fail with a diff if some files are outdated.
+make generate
+
+if ! git diff --exit-code; then
+    echo "Changes detected; diff:"
+    git diff
```

Comment:
I don't think we need to print `diff` as the `-x` of the script will do it for us.